docs/docs/code/nodejs/README.md

```diff
@@ -211,33 +341,45 @@ Sometimes you want to end your workflow early, or otherwise stop or cancel the e
 - You only want to run your workflow for users in the United States. If you receive a request from outside the U.S., you don't want the rest of the code in your workflow to run.
 - You may use the `user_id` contained in the event to look up information in an external API. If you can't find data in the API tied to that user, you don't want to proceed.
 
-**In any code step, calling the `$end()` function will end the execution of the workflow immediately.** No remaining code in that step, and no code or destination steps below, will run for the current event.
+**In any code step, calling the `$.end()` function will end the execution of the workflow immediately.** No remaining code in that step, and no code or destination steps below, will run for the current event.
 
 ```javascript
-$end();
-console.log("This code will not run, since $end() was called above it");
+defineComponent({
+  async run({ steps, $ }) {
+    $.end();
+    console.log("This code will not run, since $end() was called above it");
+  })
+});
 ```
 
-You can pass any string as an argument to `$end()`:
+You can pass any string as an argument to `$.end()`:
 
 ```javascript
-$end("Event doesn't have the correct schema");
+defineComponent({
+  async run({ steps, $ }) {
+    $.end("Event doesn't have the correct schema");
+  })
+});
 ```
 
-This message will appear in the Inspector in the **Messages** column for the event where `$end()` was called:
+This message will appear in the Inspector in the **Messages** column for the event where `$.end()` was called:
```

Comment:
You can remove this section / screenshot since we don't actually write the end message anywhere yet in Bob